core/utils/password.go

```diff
@@ -1,81 +1,38 @@
 package utils
 
 import (
-	"errors"
 	"fmt"
-	"regexp"
-	"unicode"
+	"strings"
 
+	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 )
 
 const PasswordComplexityRequirements = `
-Must be longer than 12 characters
-Must comprise at least 3 of:
-	lowercase characters
-	uppercase characters
-	numbers
-	symbols
+Must have a minimum length of 16 characters
 Must not comprise:
 	A user's API email
-	More than three identical consecutive characters
 `
 
-var (
-	lowercase = regexp.MustCompile("[a-z]")
-	uppercase = regexp.MustCompile("[A-Z]")
-	numbers   = regexp.MustCompile("[0-9]")
+const RequiredLen = 16
```

Comment:
I read "required" as something that must strictly match. e.g. password must be 16 chars long, not 15, not 17.
I would suggest to rename this to something like `MinPasswordLen`. But it's up to you of course.